sycl/test-e2e/Matrix/SG32/get_coord_int8_matB.cpp

```diff
@@ -11,8 +11,6 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
-// XFAIL: *
```

Comment:
I see xfail was removed for both gpu and cpu. Does the test pass on cpu as well?

Reply:
I don't think the pulldown of OpenCL backend that contains the fix on the CPU happened yet. So we should keep XFAIL:cpu for now.